source/linux/Foundational_Components/Kernel/Kernel_Drivers/UBIFS.rst

```diff
@@ -0,0 +1,403 @@
+UBIFS
+-----
+
+.. rubric:: Introduction
+   :name: linux-ubifs
+
+Unsorted Block Image File System is a file system specifically designed for
+flash memory devices. It's built on top of the UBI layer, which handles
+low-level flash management tasks like wear-leveling and bad block management by
+dividing it into Logical Erase Blocks (LEBs).
+
+.. rubric:: Using UBIFS on flash
+   :name: using-ubifs-on-flash
+
+Flash devices can be divided into multiple partitions, including a rootfs
+partition dedicated for the root file system. The rootfs partition can be
+formatted with the UBIFS and the volume can be mounted as the root file system
+during boot.
+
+.. rubric:: Required Software for UBI image creation
+   :name: linux-required-software-ubifs
+
+Building a UBI File System requires two applications, :command:`ubinize` and
+:command:`mkfs.ubifs`. Both are both provided by ``mtd-utils`` package.
+
+.. code-block:: console
+
+   $ sudo apt-get install mtd-utils
+
+.. rubric:: Building a UBI File System image
+   :name: linux-building-ubi-file-system
+
+.. rubric:: Step 1
+   :name: linux-building-ubi-file-system-step1
+
+Create a directory containing the root file system. It is important that the
+directory size is smaller than the ``rootfs`` partition in the flash. Users can
+download the default/tiny image for specific SoC.
+
+.. note::
+
+   Download and extract ``tisdk-default-image-*-evm.rootfs.tar.xz`` for
+   QSPI/OSPI/GPMC NAND.
+
+   Download and extract ``tisdk-tiny-image-*-evm.rootfs.tar.xz`` for QSPI/OSPI
+   NOR.
+
+The ``rootfs`` partition information
+
+   OSPI NOR
+
+   .. code-block:: console
+
+      # EVM Linux
```

Comment:
@santhosh21 , can you instead add use this in the entire file to denote EVM commands? https://github.com/TexasInstruments/processor-sdk-doc/pull/93#discussion_r1882379676 
otherwise LGTM

Reply:
Yeah, @AashvijShenai 

I've added it for all EVM commands.